app/views/curation_concerns/base/_show_actions.html.erb

```diff
@@ -14,6 +17,7 @@
     <%= link_to 'Export to Zotero', sufia.static_path('zotero'), id: 'zoteroLink', name: 'zotero', class: 'btn btn-default' %>
     <%= link_to 'Export to Mendeley', sufia.static_path('mendeley'), id: 'mendeleyLink', name: 'mendeley', class: 'btn btn-default' %>
     <!-- COinS hook for Zotero -->
-      <span class="Z3988" title="<%= export_as_openurl_ctx_kev(@presenter) %>"></span>
+    <!-- TODO: Fix issue with export_as_openurl_ctx_kev -->
```

Comment:
Is there an issue for that in GitHub, or do you think that should be part of this issue?